aws-go-s3-folder/main.go

```diff
@@ -77,3 +78,8 @@ func publicReadPolicyForBucket(bucketName pulumi.ID) (interface{}, error) {
 	})
 	return string(policy), nil
 }
+
+// ext returns the extension of the fileName.
+func ext(fileName string) string {
```

Comment:
Can this just be [`path.Ext`](https://golang.org/pkg/path/#Ext)?

Reply:
Ah, very nice.  Will update.

Reply:
Done.